src/net.cpp

```diff
@@ -140,7 +140,7 @@ bool GetLocal(CService& addr, const CNetAddr *paddrPeer)
         LOCK(cs_mapLocalHost);
         for (const auto& entry : mapLocalHost)
         {
-            int nScore = entry.second.nScore;
+            int64_t nScore = entry.second.nScore;
```

Comment:
Because of the line `nBestScore = nScore;` below, I think that `nBestScore` should be changed to `int64_t` as well.

Reply:
done - good catch! thanks!